lib/kitchen/provisioner/chef/common_sandbox.rb

```diff
@@ -92,29 +92,45 @@ def all_files_in_cookbooks
         #   kitchen root
         # @api private
         def policyfile
-          basename = config[:policyfile_path] || "Policyfile.rb"
-          File.join(config[:kitchen_root], basename)
+          if [nil, "policyfile", :policyfile].include?(config[:preferred_resolver])
```

Comment:
we have this exact same logic in chef_base.. rather than duplicate it here - why not expose a method off chef_base that common_sandbox can use?